### PR TITLE
Add readmes for SSD300/SSD512 models trained on VOC07+12

### PR DESCRIPTION
**Pull request by vanyalzr**

### CODE REVIEW

**Comment by vanyalzr** [comment]
@AlexKoff88

**Comment by eaidova** [comment]
@vanyalzr, please retarget your PR on develop branch

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/ssd300-int8-onnx-0001/description/ssd300-int8-onnx-0001.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# ssd300-int8-onnx-0001
+
+## Use Case and High-Level Description
+
+This is an SSD model with a VGG16 backbone that is designed to perform object detection. The model has been pretrained on the VOC image database (VOC07+12 dataset) and then symmetrically quantized to INT8 fixed-point precision using Neural Network Compression Framework (NNCF).
```
Comment (eizamaliev) [inline models/intel/ssd300-int8-onnx-0001/description/ssd300-int8-onnx-0001.md]: If the original public model exists in OMZ, please provide link to it

Comment (vladimir-dudnik) [reply models/intel/ssd300-int8-onnx-0001/description/ssd300-int8-onnx-0001.md]: @eizamaliev link to original model not required there

Comment (vladimir-dudnik) [reply models/intel/ssd300-int8-onnx-0001/description/ssd300-int8-onnx-0001.md]: @eizamaliev it is not required there

**Pull request merged**